internal/cli/kraft/cloud/volume/import/import.go

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+
+package vimport // "v(olume)import"; "import" is a reserved keyword
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	kraftcloud "sdk.kraft.cloud"
+	kcinstances "sdk.kraft.cloud/instances"
+	kcservices "sdk.kraft.cloud/services"
+	kcvolumes "sdk.kraft.cloud/volumes"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/initrd"
+	"kraftkit.sh/internal/cli/kraft/cloud/utils"
+	"kraftkit.sh/log"
+)
+
+type Options struct {
+	Auth  *config.AuthConfig `noattribute:"true"`
+	Token string             `noattribute:"true"`
+	Metro string             `noattribute:"true"`
+
+	Source string `local:"true" long:"source" short:"s" usage:"Path to the data source (directory, Dockerfile)" default:"."`
+	VolID  string `local:"true" long:"volume" short:"v" usage:"Identifier of an existing volume (name or UUID)"`
+}
+
+const volinitPort uint16 = 42069
+
+// Import initializes a volume by importing local data.
+func Import(ctx context.Context, opts *Options) (retErr error) {
```

Comment:
Should this be usable outside this context? If not, then:

```suggestion
func (opts *Options) Import(ctx context.Context) (retErr error) {
```

Reply:
I made the function private instead.